client/src/components/Public/entityHelpers.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import type {CompanyEntity} from '../../state'
+import type {CompanyEntity, NewEntityDetail} from '../../state'
```

Comment:
this file should ultimately disappear, with computations like these moved to selectors (or utils used by selectors, not by components)